### ISSUE TITLE
VW ID3 Adress Daten werden nicht aktualisiert -> reverseposition 

### ISSUE THREAD

**Issue by OJay99** [TA2k/ioBroker.vw-connect#322]
Hallo zusammen,
erstmal vielen Dank für den tollen Code!!

Zur Fehlerbeschreibung:
Ich logge die Daten beim losfahren und anhalten des Autos mit. 
Im IO-Broker werden die Geodaten ...position/latitudeConv und ...position/longitudeConv korrekt angezeigt.
Die Daten in ...position/address/* werden nicht korrekt angezeigt und passen nicht zu den Geodaten.
Es scheint ein Problem mit der funktion reversePosition ab Zeile 6189 zu geben. 
Leider verstehe ich zu wenig vom Programmieren, um den Fehler zu finden. 
Ich habe die Funktion kopiert und im IO-Broker zum Laufen gebracht. 
Da funktioniert die Umwandlung der Geodaten in eine Adresse wunderbar.

Vielen Dank

**Comment by TA2k** [comment]
Kannst du ein screenshot machen was du genau meinst

**Comment by OJay99** [comment]
![VW-Adapter_IOBroker_marked](https://github.com/TA2k/ioBroker.vw-connect/assets/125558003/4b1d65ee-da23-4234-8af2-37b266629d9b)

Ja, gerne.
latitudeConv -> Wert korrekt
longitudeConv -> Wert korrekt

Die Daten in "address" dagegen stimmen nicht mit den Geodaten überein. 

Irgendwie werden die "address" Daten nicht automatisch mit den Geodaten aktualisiert.

[ioBroker.vw-connect](https://github.com/TA2k/ioBroker.vw-connect/tree/master)
/main.js
Ab Zeile 6189 kommt die funktion reverseposition, die genau diese Umwandlung macht.

**Comment by TA2k** [comment]
die Umwandlung in eine Adresse passiert extern kannst du ja selber testen mit deinen werten
https://nominatim.openstreetmap.org/reverse?lat=55&lon=8&format=json

**Comment by OJay99** [comment]
Ja, danke. Das habe ich getestet. Das funktioniert mit deinem Code auch wunderbar :) Danke dafür.
Ich wollte nur sagen, dass die Daten die im IO-Broker stehen inkonsistent sind. 
Die Geodaten (Breiten- u. Längengrad) sind die tatsächlichen Werte des Standortes des Fahrzeugs. 
Die Adressdaten dagegen sind dann die Werte von dem Standort an dem das Auto beim losfahren stand. 

Aktualisierung (Update interval in minutes) steht auf 1 min. Daran kann es also nicht liegen. 
Irgendwie scheinen die Daten in "address" nicht gleichzeitig mit den Geodaten aktualisiert zu werden. 
Es stehen dann völlig falsche Werte dort...

**Comment by TA2k** [comment]
du kannst den debug log aktivieren um vielleicht irgendwelche Auffälligkeiten  zu sehen
